core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java

```diff
@@ -297,7 +295,13 @@ public String getLabelText() {
 
   @Override
   public Date getLastModified() {
-    return lastModified;
+    Date max = new Date(0);
+    for (ChangeLog c : changeLog) {
+      if (c.getTime().after(max)) {
+        max = c.getTime();
+      }
+    }
+    return max;
```

Comment:
One of the main ways this attribute is used is as the `sortBy` column on the list pools page, so it gets used as the `ORDER BY` column when selecting pools. Will this implementation of `getLastModified` be faster than the XDerivedInfo method for returning that query (aka should we be changing this for Samples, Libraries and Runs as well)?